src/main/java/game/GameEngine.java

```diff
@@ -64,7 +67,21 @@ else if (commandArgs.length == 3 && "gameplayer".equals(commandArgs[0])) {
                             System.out.println(e.getMessage());
                         }
                     }
-                } else {
+                }
+                else if(commandArgs.length== 1)
+                {
+                    if(commandArgs[0].equals("assigncountries")) {
```

Comment:
you can remove this `if` inside `else if`, and update the `else if` as `else if (commandArgs.length == 1 && commandArgs[0].equals("assigncountries"))`.

Reply:
completed